timeseries.py

```diff
@@ -0,0 +1,74 @@
+from pandas import Series, DataFrame
+from statsmodels.tsa.stattools import adfuller
+
+
+def series_train_test_split(series: Series, split=0.7) -> Series:
+    train_size = int(len(series) * split)
+    train, test = series[0:train_size], series[train_size:]
+    return train, test
+
+
+def moving_average_smoothing(series: Series, window_size: int) -> Series:
+    """ Moving average smoothing for cleaning noisy signal """
+    return series.rolling(window=window_size).mean()
+
+
+def series_differentiation(series: Series, order: int) -> Series:
+    """
+    Timeseries differentiation.Usefull for detrend series
+    """
+    if order == 1:
+        return series.diff()
+    if order == 2:
+        return series.diff().diff()
+    else:
+        print("It doesn't have sense ")
+
+
+def baseline_predictions(train: list, test: list) -> list:
+    '''
+    Return baseline predictions for naive model
+    '''
+    history = [x for x in train]
+    predictions = list()
+
+    for i in range(len(test)):
+        yhat = history[-1]
+        predictions.append(yhat)
+        # observation
+        obs = test[i]
+        history.append(obs)
+        print('>Predicted=%.3f, Expected=%3.f' % (yhat, obs))
```

Comment:
This is an old notation, kind of deprecated in the convention. You may want to consider the f{} notation instead.